cognite/client/_version.py

```diff
@@ -1,2 +1,2 @@
-__version__ = "2.24.0"
-__api_subversion__ = "V20210423"
+__version__ = "2.25.0"
+__api_subversion__ = "V20210701"
```

Comment:
The way I understand it, this variable holds the date of the earliest API change that is needed to support the SDKs functions. Is this correct? The `filter3dNodes` endpoint was added to the V1 API at this date.

Reply:
ah no, it's a different thing that is used for alpha/beta versioning of features. You just need to change `__version__`.

Reply:
```suggestion
__api_subversion__ = "V20210423"
```

Reply:
Turns out, in fact, that it is not correct :P